engine/include/cubos/engine/cubos.hpp

```diff
@@ -198,7 +201,7 @@ namespace cubos::engine
     SystemBuilder& Cubos::startupSystem(F func)
```

Comment:
This introduces a memory leak. Instead of returning `SystemBuilder&`, return `SystemBuilder`

Reply:
Fixed